meson.build

```diff
@@ -75,9 +75,20 @@ if get_option('buildtype') == 'debug'
   add_project_arguments('-DHYPRLAND_DEBUG', language: 'cpp')
 endif
 
-subdir('protocols')
-subdir('src')
-subdir('hyprctl')
-subdir('assets')
-subdir('example')
-subdir('docs')
+if get_option('pluginenv')
+  fs = import('fs')
+  globber = run_command('find', 'src', '-name', '*.hpp', check: true)
+  headers = globber.stdout().strip().split('\n')
+  foreach file : headers
+    dir = fs.parent(file)
+    fs.copyfile(file, install: true, install_dir: dir)
```

Comment:
I don't understand what this is even trying to do. You want to install all header files from the src directory to the install directory, so you are copying them to the build directory instead? Why?

Just use install_headers().

You probably also want to install a pkg-config file that plugin env software can check for.

You should double check what files are installed when running "meson install", e.g. by posting the verbose status log of meson itself. Because it's not obvious that these files are ending up in the right place right now.

Reply:
can't believe I didn't find it before, thx